src/main/scala/com/softwaremill/realworld/articles/comments/CommentsService.scala

```diff
@@ -40,13 +42,16 @@ class CommentsService(
     articlesRepository.findArticleIdBySlug(slug).someOrFail(NotFound(ArticleNotFoundMessage(slug)))
 
 object CommentsService:
+  private val ArticleAndAuthorIdsNotFoundMessage: CommentId => String = (commentId: CommentId) =>
```

Comment:
1. why not `def`s?
2. are these comments reused somehow? if not, why not simply inline them to the logInfo invocations?

Reply:
1. Generally, I relied on the code that already existed, which used def instead of val, but indeed, that needs to be changed.
2. I wanted to separate these two things and have the code written in a consistent style, but that probably needs to be changed as well.

Reply:
For such a simple thing as log messages let's just write the strings directly